declarative-schedule-doc.md

```diff
@@ -13,6 +13,7 @@
 	- [2. Enable scheduler in settings](#2-enable-scheduler-in-settings)
 	- [3. Enable scheduler in main file](#3-enable-scheduler-in-main-file)
 	- [4. Use different schedules for the same scenario](#4-use-different-schedules-for-the-same-scenario)
+	- [5. Use YAML_SCHEDULE_DEFAULT and YAML_SCHEDULE_FLOWS to YAML schedule](#5-use-default-flow-to-yaml-schedule)
```

Comment:
I would remove "to YAML schedule" from the sentence.
We don't schedule yaml file, we schedule test modules with a yaml file.

Reply:
changed.